### ISSUE TITLE
Tree - get selected IDs

### ISSUE THREAD

**Issue by peti634** [fyne-io/fyne#3828]
### Checklist

- [X] I have searched the [issue tracker](https://github.com/fyne-io/fyne/issues) for open issues that relate to the same feature, before opening a new one.
- [X] This issue only relates to a single feature. I will open new issues for any other features.

### Is your feature request related to a problem?

I would like get selected items back in Tree widget.

### Describe the solution you'd like to see.

I wrote a simple "GetSelected" function, in the Tree.go:

```
func (t *Tree) GetSelected() []TreeNodeID {
	selectedResult := make([]TreeNodeID, 0, len(t.selected))

	selectedResult = append(selectedResult, t.selected...)
	return selectedResult
}
```

I tested it, and worked.

**Comment by andydotxyz** [comment]
You can currently be told when selection happens by setting the `OnSelected` callback field.

**Comment by peti634** [comment]
Yes, I can do it, but why store it double, if the widget has this information?

**Comment by pbrown12303** [comment]
This capability is important when doing integration testing, when a side-effect of some application activity should be to select some nodes in the tree. After this action is taken, the test would like to confirm that the appropriate tree nodes have been selected.
